charts/kcp/values.yaml

```diff
@@ -43,15 +49,7 @@ kcpFrontProxy:
     enabled: false
     className: ""
   certificate:
-    issuerSpec:
-      acme:
-        server: https://acme-v02.api.letsencrypt.org/directory
-        privateKeySecretRef:
-          name: kcp-front-proxy-issuer-account-key
-        solvers:
-        - http01:
-            ingress:
-              serviceType: ClusterIP
+    issuerSpec: {selfSigned: {}}
```

Comment:
This part is complicated. As this is treated like a map, helm merges values here. 
I think the solution would be to keep this empty and ask users explicitly provide value via `values.yaml`
Having default in `values.yaml` as its currently is makes it impossible to override. 

It might be possible to workaround this somehow, but I didn't find a quick way. Need more experimentation .